### DISCUSSION TITLE
Q/A: Do you see yourself as a developer, consumer and/or supporter of z/OS Open Source?

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#22] [Guild Feedback]
How do you see yourself getting involved in the community (supporter, developer, consumer, other ?)

If you're interested in developing:
- do you currently work on z/OS or have access to a z/OS system? 
- If not, are you interested in getting access to a system
- if you have access to a z/OS system, do you use Unix System Services or MVS Batch?
  - if so, do you have access to github from z/OS?

**Comment by MikeFultonDev** [comment]
Developer and Consumer. I have access to a system

**Comment by IgorTodorovskiIBM** [comment]
Developer and Consumer. I have access to a system

**Comment by jshimoda9323** [comment]
Also developer and consumer, and have access to a system.

**Comment by MikeFultonDev** [comment]
done! we have this as part of our periodic polls on the guild calls now
